packages/orchestration/src/orchestration.js

```diff
@@ -216,7 +246,9 @@ const prepareOrchestration = (zone, createChainAccount) =>
           await E(port)
             .connect(remoteConnAddr, chainAccount.connectionHandler)
             // XXX if we fail, should we close the port (if it was created in this flow)?
-            .catch(e => Fail`Failed to create ICA connection: ${bare(e)}`);
+            .catch(e => {
+              throw Fail`Failed to create ICA connection: ${bare(e)}`;
+            });
```

Comment:
did this change anything or purely stylistic?

Reply:
Removed